scripts/sync_packages.sh

```diff
@@ -1,35 +1,41 @@
 #!/bin/bash
-robot_name="${HOSTNAME//-b1}"
```

Comment:
yes, and/or use nmap to find the robot pcs:

`client_list=$(nmap --unprivileged $IP-98 --system-dns | grep report | awk '{print $6}' | sed 's/(//g;s/)//g') `